mod-audit-server/src/main/java/org/folio/rest/impl/CirculationLogsService.java

```diff
@@ -38,11 +43,13 @@ public void getAuditDataCirculationLogs(String query, int offset, int limit, Str
                 .withLogRecords(results)
                 .withTotalRecords(reply.result().getResultInfo().getTotalRecords()))));
           } else {
+            LOGGER.warn("Failed to retrieve audit data circulation logs: {}", reply.cause().getMessage());
             asyncResultHandler.handle(succeededFuture(GetAuditDataCirculationLogsResponse
               .respond400WithApplicationJson(buildError(HTTP_BAD_REQUEST.toInt(), reply.cause().getLocalizedMessage()))));
           }
         }))
       .exceptionally(throwable -> {
+        LOGGER.warn("Exception occurred while getting audit data circulation logs: {}", throwable.getMessage());
```

Comment:
should it be getMEssage or getLocalizedMessage ?

Reply:
It should have been getLocalizedMessage. I have fixed it.